<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>masterCashierClass</class>
 <widget class="QMainWindow" name="masterCashierClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>masterCashier</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#wgt_title{
height:80px;
background-color:#081E4B;
}
#wgt_left{
width:100px;
background-color:#142B59;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="wgt_title" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#btn_min{
border-image:url(:/masterCashier/Resources/tab_min.png);
}
#btn_min:pressed{
margin:1px;
}
#btn_max{
border-image:url(:/masterCashier/Resources/tab_max.png);
}
#btn_max:pressed{
margin:1px;
}
#btn_close{
border-image:url(:/masterCashier/Resources/tab_close.png);
}
#btn_close:pressed{
margin:1px;
}
#btn_settings{
border-image:url(:/masterCashier/Resources/tab_set.png);
}
#btn_settings:pressed{
margin:1px;
}
#btn_switch{
border-image:url(:/masterCashier/Resources/tab_change.png);
}
#btn_switch:pressed{
margin:1px;
}
#btn_icon{
border-image:url(:/masterCashier/Resources/tab_more.png);
}
#btn_icon:pressed{
margin:1px;
}
#lbl_icon{
border-radius:15px;
}
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_logo">
         <property name="minimumSize">
          <size>
           <width>650</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>650</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-family:&quot;微软雅黑&quot;;
color:#FFFFFF;
font-size:20px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>579</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <widget class="QLabel" name="lbl_icon">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image:url(:/masterCashier/Resources/login/head.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_icon">
             <property name="minimumSize">
              <size>
               <width>8</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>8</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_switch">
           <property name="minimumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_settings">
           <property name="minimumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:#203A70;</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_min">
           <property name="minimumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_max">
           <property name="minimumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close">
           <property name="minimumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wgt_left" native="true">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#btn_shift{
font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:rgba(255,255,255,0.9);
background:#203B73;
border:none;
}
#btn_shift:pressed{
margin:1px;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="classifyButton" name="btn_qySupmarket">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>24</height>
            </size>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoRepeatDelay">
            <number>300</number>
           </property>
           <property name="autoRepeatInterval">
            <number>100</number>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>583</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_shift">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>交班</string>
           </property>
           <property name="icon">
            <iconset resource="masterCashier.qrc">
             <normaloff>:/masterCashier/Resources/shifting.png</normaloff>:/masterCashier/Resources/shifting.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="wgt_right">
        <widget class="pageQySupmarketWgt" name="page_qySupmarket"/>
        <widget class="handOverShiftWgt" name="page_shift"/>
        <widget class="PageSettingsWidget" name="page_settings"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>classifyButton</class>
   <extends>QPushButton</extends>
   <header>classifyButton.h</header>
  </customwidget>
  <customwidget>
   <class>pageQySupmarketWgt</class>
   <extends>QWidget</extends>
   <header>pageQySupmarketWgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>handOverShiftWgt</class>
   <extends>QWidget</extends>
   <header>handOverShiftWgt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageSettingsWidget</class>
   <extends>QWidget</extends>
   <header>PageSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="masterCashier.qrc"/>
 </resources>
 <connections/>
</ui>
